src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -92,11 +93,11 @@
   </KTable>
 </template>
 
-<script lang="ts" setup>
+<script lang="ts" setup generic="Row extends { id: string }">
```

Comment:
**Comment**: This enforces `items` to always have an `id`. While Kuma resources traditionally don’t have an ID and are instead identified using their name, this isn’t guaranteed to always be true. Enforcing an ID here should be more robust overall and allow for both resources identified by ID or name.